src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -416,13 +455,10 @@ export default function DownloadDialog({
   }
 
   const readyToInstall =
-    installPath &&
-    gameInstallInfo?.manifest?.download_size &&
-    !gettingInstallInfo &&
-    validFlatpakPath
+    installPath && diskSize && !gettingInstallInfo && validFlatpakPath
```

Comment:
looks like this `diskSize` not defaults to 0 by before it defaulted to `false` while loading

that seems to be causing this later in the component

<img width="119" alt="Screenshot 2023-09-30 at 3 03 06 PM" src="https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/46e7a685-f655-4113-b0cf-a9170c6f5ba9">

Reply:
ah, merging leftovers